Client/core/CCore.cpp

```diff
@@ -1337,9 +1339,19 @@ void CCore::DoPostFramePulse()
     GetGraphStats()->Draw();
     m_pConnectManager->DoPulse();
 
-    static auto discord = g_pCore->GetDiscord();
-    if (discord && discord->IsDiscordRPCEnabled())
-        discord->UpdatePresence();
+    // Update Discord Rich Presence status
+    if (const long long ticks = GetTickCount64_(); ticks > m_timeDiscordAppLastUpdate + TIME_DISCORD_UPDATE_RICH_PRESENCE_RATE)
+    {
+        if (static const auto discord = g_pCore->GetDiscord(); discord && discord->IsDiscordRPCEnabled())
```

Comment:
static variable here can lead to a fault. If g_pCore->GetDiscord() at some time return null, discord will still returning a pointer.

Reply:
It was a mistake from previous versions. Already fixed.